packages/box/CHANGELOG.md

```diff
@@ -3,13 +3,13 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-# [1.2.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/box@1.1.0...@littlespoon/box@1.2.0) (2021-10-28)
+## [1.2.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/box@1.1.0...@littlespoon/box@1.2.0) (2021-10-28)
```

Comment:
Made headings consistent (h2) here and below since these are rendered in Storybook